src/wagtailmedia/templates/wagtailmedia/chooser/chooser-next.html

```diff
@@ -0,0 +1,90 @@
+{% load i18n wagtailadmin_tags %}
+{% trans "Choose a media file" as  choose_str %}
+{% include "wagtailadmin/shared/header.html" with title=choose_str merged=1 icon="media" %}
+
+{# TODO: drop data-wmtabs and the tabs.js include when a better tabs framework is in place #}
+<div class="w-tabs" data-tabs data-tabs-animate data-wm-tabs>
```

Comment:
```suggestion
<div class="w-tabs" data-tabs data-tabs-animate data-tabs-disable-url data-wm-tabs>
```
This is analog to the image's/document's chooser.
Currently, without the `data-tabs-disable-url`, if you open the media chooser, click "Upload Audio", `#tab-upload-audio` gets added to the browser location, if you now reload the page, and then reopen the media chooser, in my case, I see blank areas around the upload form, so I have to scroll down a bit. So the chooser without this option seems to be a bit buggy.